testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -392,25 +420,32 @@ def test_resid_single(self, universe):
 
     def test_same_coordinate(self, universe):
         """Test the 'same ... as' construct (Issue 217)"""
-        sel = universe.select_atoms("same x as bynum 1 or bynum 10")
-        assert_equal(len(sel), 12,
-                     "Found a wrong number of atoms with same x as ids 1 or 10")
-        target_ids = np.array([0, 8, 9, 224, 643, 3515,
-                               11210, 14121, 18430, 25418, 35811, 43618])
-        assert_equal(sel.indices, target_ids,
-                     "Found wrong atoms with same x as ids 1 or 10")
```

Comment:
Does the implicit OR work here? Ie “index 0 9”

Reply:
It does, I've added those cases to the list of strings being tested, and also added index to `test_range_selections` in `TestImplicitOr`.